spec/chewy/type/adapter/active_record_spec.rb

```diff
@@ -427,7 +437,10 @@ def delete_already?
         subject { described_class.new(Country.joins(:cities)) }
 
         specify { expect(subject.import_fields(Country.where('rating < 2'))).to match([contain_exactly(1, 2)]) }
-        specify { expect(subject.import_fields(Country.where('rating < 2'), fields: [:rating])).to match([contain_exactly([1, 0], [2, 1])]) }
+        specify do
+          expect(subject.import_fields(Country.where('rating < 2'),
+                                       fields: [:rating])).to match([contain_exactly([1, 0], [2, 1])])
+        end
```

Comment:
I would break line with .to in next line here and in the next 5 changes of this file